src/mission_ui.cpp

```diff
@@ -71,9 +74,9 @@ class mission_ui_impl : public cataimgui::window
         mission_ui_tab_enum selected_tab = mission_ui_tab_enum::ACTIVE;
         mission_ui_tab_enum switch_tab = mission_ui_tab_enum::num_tabs;
 
-        size_t window_width = str_width_to_pixels( TERMX ) / 2;
-        size_t window_height = str_height_to_pixels( TERMY ) / 2;
-        size_t table_column_width = window_width / 2;
+        float window_width = std::max( ImGui::GetMainViewport()->Size.x / 2, minimum_window_width );
+        float window_height = std::max( ImGui::GetMainViewport()->Size.y / 2, minimum_window_height );
```

Comment:
Consider using `std::clamp` to ensure you don't overflow out of display bounds.

Reply:
I did initially try std:clamp, but it didn't seem to work due to float. I tried to work aaround it, but eventually settled on std::max

Reply:
Think I got it now... Prepping new commit
Don't code at 3AM lol